content/release-notes/2022-10-25.md

```diff
@@ -0,0 +1,16 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Compute: Add support of PostgreSQL 15.0 and its PostgreSQL extensions.
+For information about supported extensions, see [Available PostgreSQL extensions](https://neon.tech/docs/reference/compatibility/#available-postgresql-extensions).
+- Compute: Fix creation of databases with trailing spaces in database name.
```

Comment:
Can you describe what this issue was and where it occurred? Did the user supply a database name with trailing in the database creation dialog that caused some problem? What issue did it cause?

Reply:
> Did the user supply a database name with trailing in the database creation dialog that caused some problem?

Yes. here is the description
https://github.com/neondatabase/neon/pull/2652

Reply:
Thanks. Suggested text below.